processor/apmprocessor/span.go

```diff
@@ -34,6 +34,11 @@ func (sp *spanProcessor) processTraces(_ context.Context, td ptrace.Traces) (ptr
 				span := scopeSpan.Spans().At(k)
 				if parsedTable, parsed := sp.sqlparser.ParseDbTableFromSpan(span); parsed {
 					span.Attributes().PutStr(DbSQLTableAttributeName, parsedTable)
+					if dbSystem, dbSystemPresent := span.Attributes().Get(DbSystemAttributeName); dbSystemPresent && dbSystem.AsString() == "redis" {
```

Comment:
I think it'd be cleaner to add that logic to the sql parser

Reply:
Hmm.  It doesn't feel _quite_ right to me.  I envisioned that class as something that parsed the table name from sql statements.  This is a little different.  It could be moved to a general "sql utility" style of class.

Reply:
Right. What doesn't feel right is to have redis-related logic in both the processor and the parser. So perhaps we should take that logic out of the parser and put it in a more generic "database" utility class - which would make sense, since redis has little to do with sql.